### PR TITLE
Added new shadowserver files to _config.py

### PR DESCRIPTION
**Pull request by mateodurante**
Added new shadowserver event types and files to _config.py:
- Accesible AMQP
- Device Identification
- SSL Poodle IPv6

### CODE REVIEW

File: intelmq/bots/parsers/shadowserver/_config.py [hunk 1]
```diff
@@ -2903,10 +2903,78 @@ def scan_exchange_identifier(field):
...
+        ('extra.', 'locales', validate_to_none),
+    ],
+    'constant_fields': {
+        'classification.taxonomy': 'other',
+        'classification.type': 'other',
+        'classification.identifier': 'accessible-amqp',
+    }
+}
+
+# https://www.shadowserver.org/what-we-do/network-reporting/accessible-amqp-report/
```
Comment (sebix) [inline intelmq/bots/parsers/shadowserver/_config.py]:
```suggestion
# https://www.shadowserver.org/what-we-do/network-reporting/device-identification-report/
```

File: intelmq/bots/parsers/shadowserver/_config.py [hunk 2]
```diff
@@ -2903,10 +2903,78 @@ def scan_exchange_identifier(field):
...
+        ('extra.', 'cluster_name', validate_to_none),
+        ('extra.', 'platform', validate_to_none),
+        ('extra.', 'product', validate_to_none),
+        ('extra.', 'product_version', validate_to_none),
+        ('extra.', 'mechanisms', validate_to_none),
+        ('extra.', 'locales', validate_to_none),
+    ],
+    'constant_fields': {
+        'classification.taxonomy': 'other',
+        'classification.type': 'other',
```
Comment (sebix) [inline intelmq/bots/parsers/shadowserver/_config.py]:
```suggestion
        'classification.taxonomy': 'vulnerable',
        'classification.type': 'vulnerable-system',
```

File: intelmq/bots/parsers/shadowserver/_config.py [hunk 3]
```diff
@@ -2903,10 +2903,78 @@ def scan_exchange_identifier(field):
...
+        ('extra.', 'naics', invalidate_zero),
+        ('extra.', 'sic', invalidate_zero),
+        ('extra.', 'sector', validate_to_none),
+        ('extra.', 'device_vendor', validate_to_none),
+        ('extra.', 'device_type', validate_to_none),
+        ('extra.', 'device_model', validate_to_none),
+    ],
+    'constant_fields': {
+        'classification.taxonomy': 'other',
+        'classification.type': 'other',
```
Comment (sebix) [inline intelmq/bots/parsers/shadowserver/_config.py]: We can also use `undetermined` here. The RSIT definition of undetermined is `The categorisation of the incident is unknown/undetermined.`

**Review by sebix — CHANGES_REQUESTED** [review]
Looks good semantically, if someone can verify the validity of the mapping and also the file names against real-world data, it's good to merge. Tests would be cool, but not strictly required.

File: intelmq/bots/parsers/shadowserver/_config.py (hunk 3 above)
Comment (mateodurante) [reply intelmq/bots/parsers/shadowserver/_config.py]: Yes, undetermined may be more appropriate since it is only informational

**Review by mateodurante — COMMENTED** [review]
I believe I have made the proposed changes, if not, please let me know

**Review by sebix — APPROVED** [review]

**Review by waldbauer-certat — APPROVED** [review]

**Comment by sebix** [comment]
Thanks again @mateodurante 

Documentation update in https://github.com/certtools/intelmq/pull/2135

**Pull request merged**